types/follow-redirects/index.d.ts

```diff
@@ -0,0 +1,118 @@
+// Type definitions for follow-redirects 1.5
+// Project: https://github.com/olalonde/follow-redirects
+// Definitions by: Emily Klassen <https://github.com/forivall>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+/// <reference types="node" />
```

Comment:
Are there any types that you're using from node? I think this can be removed.

Reply:
http, https and stream require node to be included: 

When running lint:
```
ERROR: 7:27  expect  TypeScript@next compile error: 
Cannot find module 'http'.
ERROR: 8:28  expect  TypeScript@next compile error: 
Cannot find module 'https'.
ERROR: 9:26  expect  TypeScript@next compile error: 
Cannot find module 'stream'.
```